java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/AzureEventHubsSourceTask.java

```diff
@@ -87,4 +94,21 @@ public void stop() {
     ofNullable(eventHubsKafkaConsumerController)
         .ifPresent(consumerController -> consumerController.close(closeTimeout));
   }
+
+  /**
+   * Returns output topics (if specified in config) or throws {@link ConfigException}.
+   *
+   * @param azureEventHubsSourceConfig task configuration
+   * @return output topics list
+   */
+  private List<String> getOutputTopicsFromConfig(
+      AzureEventHubsSourceConfig azureEventHubsSourceConfig) {
+    Kcql kcql = KcqlConfigPort.parseMultipleKcqlStatementsPickingOnlyFirst(
+        azureEventHubsSourceConfig.getString(AzureEventHubsConfigConstants.KCQL_CONFIG));
+    List<String> outputTopics = Arrays.stream(kcql.getTarget().split(",")).collect(Collectors.toList());
+    if (outputTopics.isEmpty()) {
+      throw ExceptionProviders.OUTPUT_TOPIC_CONFIG_EXCEPTION_SUPPLIER.get();
```

Comment:
i don't think throwing exception for empty is adding value.

Reply:
Removed, I'll just leave it to KCQL parsing then.